x/group/spec/02_state.md

```diff
@@ -0,0 +1,80 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `group` module uses the `orm` package which provides table storage with support for
+primary keys and secondary indexes. `orm` also defines `Sequence` which is a persistent unique key generator based on a counter that can be used along with `Table`s.
+
+Here's the list of tables and associated sequences and indexes stored as part of the `group` module.
+
+## Group Table
+
+The `groupTable` stores `GroupInfo`: `0x0 | []byte(group_id) -> ProtocolBuffer(GroupInfo)`
+
+### groupSeq
+
+The value of `groupSeq` is incremented when creating a new group and corresponds to the new `GroupId`: `0x1 | 0x1 -> BigEndian`
+
+### groupByAdminIndex
+
+`groupByAdminIndex` allows to retrieve groups by admin address using prefix `0x2`.
```

Comment:
I think it would be interesting for readers to know how the By*Index translates into key/value pairs `0x2 | ??? -> ???` (I would like to know that myself actually)

Reply:
done